reports/report.md

```diff
@@ -6611,3 +6619,752 @@ State variables that are not updated following deployment should be declared con
+
+- Found in src/ArbitraryTransferFrom.sol [Line: 11](../tests/contract-playground/src/ArbitraryTransferFrom.sol#L11)
+
+	```solidity
+	    constructor(IERC20 token) {
+	```
+
+- Found in src/AssertStateChange.sol [Line: 8](../tests/contract-playground/src/AssertStateChange.sol#L8)
+
+	```solidity
+	    function bad() public {
+	```
+
+- Found in src/AssertStateChange.sol [Line: 12](../tests/contract-playground/src/AssertStateChange.sol#L12)
+
+	```solidity
+	    function good() public {
+	```
+
+- Found in src/CacheArrayLength.sol [Line: 23](../tests/contract-playground/src/CacheArrayLength.sol#L23)
+
+	```solidity
+	    function c3() external {
+	```
+
+- Found in src/CacheArrayLength.sol [Line: 32](../tests/contract-playground/src/CacheArrayLength.sol#L32)
+
+	```solidity
+	    function c4() external {
+	```
+
+- Found in src/Casting.sol [Line: 30](../tests/contract-playground/src/Casting.sol#L30)
+
+	```solidity
+	    function unsafeUintCasting() external {
+	```
+
+- Found in src/Casting.sol [Line: 68](../tests/contract-playground/src/Casting.sol#L68)
+
+	```solidity
+	    function unsafeIntCasting() external {
+	```
+
+- Found in src/Casting.sol [Line: 106](../tests/contract-playground/src/Casting.sol#L106)
+
+	```solidity
+	    function unsafeBytes32Casting() external {
+	```
+
+- Found in src/ConstFuncChangeState.sol [Line: 8](../tests/contract-playground/src/ConstFuncChangeState.sol#L8)
+
+	```solidity
+	    function changeState() public view returns (uint) {
+	```
+
+- Found in src/ConstFuncChangeState.sol [Line: 14](../tests/contract-playground/src/ConstFuncChangeState.sol#L14)
+
+	```solidity
+	    function changeState2() public returns (uint) {
+	```
+
+- Found in src/CostlyOperationsInsideLoops.sol [Line: 9](../tests/contract-playground/src/CostlyOperationsInsideLoops.sol#L9)
+
+	```solidity
+	    function bad() external {
+	```
+
+- Found in src/CostlyOperationsInsideLoops.sol [Line: 15](../tests/contract-playground/src/CostlyOperationsInsideLoops.sol#L15)
+
+	```solidity
+	    function good() external {
+	```
+
+- Found in src/Counter.sol [Line: 7](../tests/contract-playground/src/Counter.sol#L7)
+
+	```solidity
+	    function setNumber(uint256 newNumber) public {
+	```
+
+- Found in src/Counter.sol [Line: 11](../tests/contract-playground/src/Counter.sol#L11)
+
+	```solidity
+	    function increment() public {
+	```
+
+- Found in src/Counter.sol [Line: 16](../tests/contract-playground/src/Counter.sol#L16)
+
+	```solidity
+	    function callIncrement() external {
+	```
+
+- Found in src/Counter.sol [Line: 21](../tests/contract-playground/src/Counter.sol#L21)
+
+	```solidity
+	    function incrementByTwoMagic() external {
+	```
+
+- Found in src/Counter.sol [Line: 28](../tests/contract-playground/src/Counter.sol#L28)
+
+	```solidity
+	    function incrementByTwoConstant() external {
+	```
+
+- Found in src/DeadCode.sol [Line: 7](../tests/contract-playground/src/DeadCode.sol#L7)
+
+	```solidity
+	    constructor(uint256 _value) {
```

Comment:
We should omit constructors from this detector @TilakMaddy .

Reply:
Done!